### PR TITLE
gnn network train infer

### PR DESCRIPTION
**Pull request by hutuxian**

### CODE REVIEW

File: fluid/PaddleRec/gnn/network.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
+import paddle
```
Comment (guru4elephant) [inline fluid/PaddleRec/gnn/network.py:15]: You should add License on the head of the files

File: fluid/PaddleRec/gnn/network.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    seq_fc = layers.fc(
+        input=seq,
+        name="seq_fc",
+        size=hidden_size,
+        bias_attr=False,
+        act=None,
+        num_flatten_dims=2,
+        param_attr=fluid.ParamAttr(initializer=fluid.initializer.Uniform(
+            low=-stdv, high=stdv)))  #[batch_size, -1(seq_max), h]
+    last_fc = layers.fc(input=last,
```
Comment (gavin1332) [inline fluid/PaddleRec/gnn/network.py]: you'd better make the parameter format of fc op consistent

File: fluid/PaddleRec/gnn/network.py [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+                        name="last_fc",
+                        size=hidden_size,
+                        bias_attr=False,
+                        act=None,
+                        num_flatten_dims=1,
+                        param_attr=fluid.ParamAttr(
+                            initializer=fluid.initializer.Uniform(
+                                low=-stdv, high=stdv)))  #[bathc_size, h]
+    b = layers.create_parameter(
```
Comment (gavin1332) [inline fluid/PaddleRec/gnn/network.py:146]: move parameter creation just before it is used, or gather up them to the front

File: fluid/PaddleRec/gnn/network.py [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+        shape=[hidden_size],
+        dtype='float32',
+        default_initializer=fluid.initializer.Constant(value=0.0))  #[h]
+
+    seq_fc_t = layers.transpose(
```
Comment (gavin1332) [inline fluid/PaddleRec/gnn/network.py]: in consideration of efficiency, the target shape should be [-1, batch_size, h]

**Review by gavin1332 — COMMENTED** [review]
some code is need update

**Review by gavin1332 — APPROVED** [review]

**Review by gavin1332 — APPROVED** [review]

**Pull request merged**